### ISSUE TITLE
....implements HVACMode(s): auto, heat, off and therefore implicitly supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature

### ISSUE THREAD

**Issue by Chaoscontrol** [asantaga/wiserHomeAssistantPlatform#435]
Warning in the log. Haven't seen this one before. 

`Entity None (<class 'custom_components.wiser.climate.WiserRoom'>) implements HVACMode(s): auto, heat, off and therefore implicitly supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature.`
```
Integration v3.4.2.
HA:
Core
2024.1.6
Supervisor
2023.12.1
Operating System
11.4
Frontend
20240104.0
HubR, with single channel, on a combi boiler without OpenTherm. Firm 3.14.0.
```

**Comment by LGO44** [comment]
I have the same issue

**Comment by msp1974** [comment]
I know @Chaoscontrol says they are on HA 2024.1.6 but are you not on 2024.2.0?  Looking in the developer blog, this is a deprecation in 2024.2 and needs the supported features updating in the code.  Will add to the list but it should not affect any functionality until 2025.1.

**Comment by LGO44** [comment]
> I know @Chaoscontrol says they are on HA 2024.1.6 but are you not on 2024.2.0? Looking in the developer blog, this is a deprecation in 2024.2 and needs the supported features updating in the code. Will add to the list but it should not affect any functionality until 2025.1.

I am on HA 2024.2.0

**Comment by Chaoscontrol** [comment]
Right now I am too on HA 2024.2.0. 

Actually sorry, in my report I copy pasted the system info from another post. When this happened was right after updating to 2024.0.

**Comment by andyblac** [comment]
getting the same

```
Logger: homeassistant.components.climate
Source: components/climate/__init__.py:354
Integration: Climate (documentation, issues)
First occurred: 11:06:13 (6 occurrences)
Last logged: 11:06:13

Entity None (<class 'custom_components.wiser.climate.WiserRoom'>) implements HVACMode(s): auto, heat, off and therefore implicitly supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature. Please create a bug report at https://github.com/asantaga/wiserHomeAssistantPlatform/issues
```

```
Core: 2024.2.4
Supervisor: 2024.02.0
Operating System: 12.0
Frontend: 20240207.1
```

**Comment by Chaoscontrol** [comment]
Just to add some info. This error shows once every time I restart HA. Then it never shows again.

**Comment by msp1974** [comment]
Resolved in v3.4.3 - just released
